src/components/mobileservices/validator/RuleSetValidator.js

```diff
@@ -51,6 +51,7 @@ export class RuleSetValidator {
       return false;
     }
     let hasErrors = false;
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Do we still need this?

Reply:
Nope, We do not. I missed these two the first time around. They should be changed now.